xml/System.Net.Security/CipherSuitesPolicy.xml

```diff
@@ -39,9 +39,19 @@
         <Parameter Name="allowedCipherSuites" Type="System.Collections.Generic.IEnumerable&lt;System.Net.Security.TlsCipherSuite&gt;" />
       </Parameters>
       <Docs>
-        <param name="allowedCipherSuites">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="allowedCipherSuites">The collection of cipher suites allowed in this policy for negotiation.</param>
```

Comment:
should we also have a corresponding change updating comments in corefx?

Reply:
It's not required, and it's not part of the effort to backport the documentation to triple slash in code. The source of truth is dotnet-api-docs.

Reply:
@krwq/@jozkee We might want to update the `.tt` files to include the summary value for the enum members, so when we run it in the future we can just use Carlos' tooling to merge the corefx comments with the placeholder docs here.